File: app/src/main/java/com/lenar/fitforlife/screens/SplashActivity.java
package com.lenar.fitforlife.screens;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;

import com.lenar.fitforlife.R;

public class SplashActivity extends AppCompatActivity {

    private static final long WAITING_TIME = 2500;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                final Intent mainIntent = new Intent(SplashActivity.this, MainActivity.class);
                SplashActivity.this.startActivity(mainIntent);
                SplashActivity.this.finish();
            }
        }, WAITING_TIME);
    }
}

File: app/src/main/java/com/nilden/fitforlife/screens/SplashActivity.kt
package com.nilden.fitforlife.screens

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity

import com.nilden.fitforlife.R

class SplashActivity : AppCompatActivity() {

    private val WAITING_TIME: Long = 2500

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        Handler().postDelayed({
            val mainIntent = Intent(this@SplashActivity, MainActivity::class.java)
            this@SplashActivity.startActivity(mainIntent)
            this@SplashActivity.finish()
        }, WAITING_TIME)
    }
}
